### ISSUE TITLE
Display Chain Stats number range

### ISSUE THREAD

**Issue by nccreang** [prmiles/pymcmcstat#79]
Print out of the chain stats at the end of sampling does not display values below ~1e-5.  It would be nice if the number cut off for display can be extended to about 1e-20 in order to display smaller numbers (I am working with numbers between 1e-11 and 1e-13).  I know this same issue has been fixed in the print out of the sampling parameters but has not been fixed for the display after sampling. 
![Screen Shot 2020-01-06 at 4 58 07 PM](https://user-images.githubusercontent.com/48766292/71852005-c6b96f80-30a5-11ea-9afb-6c48b96fb5a8.png)

**Comment by prmiles** [comment]
@nccreang That is a good point!  I cannot implement a solution for the display quickly, but there is a work around for you to get those numbers with higher precision.  The [`chainstats`](https://github.com/prmiles/pymcmcstat/blob/master/pymcmcstat/chain/ChainStatistics.py#L21) function can optionally return the statistics in a dictionary by passing the argument `returnstats=True`.  So, for your particular problem, you could get the mean values for A1 and A2 as follows:

```python
stats = chainstats(chain, results, returnstats=True)
mean=stats['mean']
A1_mean = mean[2] # Average for A1
A2_mean = mean[3] # Average for A2
```

Note, I'm not entirely sure at which point I added this feature to the package, so I would recommend updating to the latest version (1.9.0).

I'm going to leave the issue open because I would like to implement what you suggested; however, I do not have any time at the moment.  Please let me know if the method I described above provides sufficient information.

**Comment by prmiles** [comment]
@nccreang I finally got a chance to fix this.  The chain stats display should now behave similarly to the sampling parameters.  Note, this fix is included in v1.9.1.
